File: Javascript/4_function_4/callback.js
function simple(a, b, callback) {
    return callback(a, b);
}
function add(a, b) {
    return a + b;
}
function sub(a, b) {
    return a - b;
}
function mul(a, b) {
    return a * b;
}
function div(a, b) {
    return a / b;
}
console.log(simple(10, 20, add));
console.log(simple(10, 20, sub));
console.log(simple(10, 20, mul));
console.log(simple(10, 20, div));

File: Python practice/12 function/callback.py
def simple(a, b, func):
    return func(a, b)

def add(a, b):
    return a + b

def sub(a, b):
    return a - b

def mul(a, b):
    return a * b

def div(a, b):
    return a / b

if __name__ == "__main__":
    print(simple(10, 20, add))
    print(simple(10, 20, sub))
    print(simple(10, 20, mul))
    print(simple(10, 20, div))